app/graphql/schema/queries/all-customer-query-tickets.js

```diff
@@ -0,0 +1,80 @@
+const cosmos = require('../../../cosmos')
```

Comment:
This still feels a bit overly complex to me. I was wondering if we could add an extra field something like `originalQuery` as a boolean that way we could identify the original querys and the responses. Then we should be able to do a query just for the original queries.

What do you think to this?

However as this is a spike I think we can leave it as is for now and look to refactor it later once we know more about the queries data structure

Reply:
Ah I was thinking of another field we could use to distinguish the original queries from the responses but was stuck on what to go with, this is a good shout though! Will work to add it in.